tools/create_release.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, repo: T.Optional[str], token: T.Optional[str], tag: str):
             self.create_wrap_file()
 
     def read_wrap(self):
-        filename = Path('subprojects', self.name + '.wrap')
+        filename = Path('subprojects', f"{self.name}.wrap")
```

Comment:
flynt appears to be converting all single quotes to double quotes for inexplicable reasons...

Reply:
ruff does too. I don't know either.

Reply:
Well, can you switch them back? ;)

By the way, ruff has

```toml
[format]
quote-style = "single"
```

No clue about flynt...

Reply:
interestingly, flynt does not always convert to "". I ran this somewhere else and it kept the ''.